.travis.yml

```diff
@@ -145,7 +147,7 @@ jobs:
     # -------------------------------------------------------------------------
     - stage: Deployment
       name: Deploy to docs.kuzzle.io
-      if: branch = master
+      if: branch = master AND (tag IS present)
```

Comment:
does that mean that we have to create a new version evrytime we need to release the documentation?

Reply:
yes

Reply:
Then I'm blocking this PR until we discuss this internally (I already started a thread on slack), because as our current release process goes, this is too cumbersome and impractical.